### DISCUSSION TITLE
Bash script to update Cloudlog

### DISCUSSION THREAD

**Discussion by M5JFS** [magicbug/Cloudlog#868] [Q&A] [answered]
I suspect there is an issue with my setup, the script runs ok but is always reports  **Already up to date.**   I'm not convinced that it is.

How can I confirm the level of my Cloudlog installation and what could be the reason for this behaviour if it turns out not to be updating?

**Comment by AndreasK79** [comment]
If you have USA County under General in the QSO dialog, you are updated. That was added to the code January 31st.

I have no idea why the script doesn't work. Perhaps there are some logs you can check.

**Reply by M5JFS** [reply]
Thanks Andreas, - no I don't have USA County in my QSO dialog General tab ☹️

Time to start digging deeper, if i can't fix it, then maybe I will just re-install with the latest code.  Trouble is, while I can do basic stuff with Linux, problem determination and fixing is a bit beyond me.

**Reply by magicbug** [reply]
Don't go to wild yet, move into the Cloudlog folder and do "git pull" see what happens

**Reply by magicbug** [reply]
You can also do git status and see what's returned

**Reply by M5JFS** [reply]
> Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
You have new mail.
Last login: Sat Feb  6 05:27:08 2021 from 192.168.0.180
pi@onyx:~ $ ls
Cloudlog  cloudlog.sh
pi@onyx:~ $ cd Cloudlog/
pi@onyx:~/Cloudlog $ ld
ld: no input files
pi@onyx:~/Cloudlog $ ls
application  CloudLog.png  index.php  README.md   updates
assets       favicon.ico   install    robots.txt  uploads
backup       images        LICENSE    system
pi@onyx:~/Cloudlog $ git pull
Already up to date.
pi@onyx:~/Cloudlog $ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
pi@onyx:~/Cloudlog $ ^C
pi@onyx:~/Cloudlog $

**Reply by magicbug** [reply]
But very helpful ok let's try

git log

Should return a nice list of commits, I've a feeling your up to date

**Reply by M5JFS** [reply]
pi@onyx:~/Cloudlog $ git log
commit 3bbc0b4736f8cb13c535ba6c1a93901a1fa9ce1c (HEAD -> master, origin/master, origin/HEAD)
Merge: 89d5794 75a9684
Author: Peter Goodhall <magicbug@users.noreply.github.com>
Date:   Wed Feb 3 16:35:46 2021 +0000

    Merge pull request #865 from oh1jw/patch-1

    Update general_words_lang.php for Finnish

commit 75a9684cd3946577cb6ac9ec454404beea319d5c
Author: oh1jw <77230101+oh1jw@users.noreply.github.com>
Date:   Wed Feb 3 15:21:45 2021 +0200

    Update general_words_lang.php

    updated translation

commit 89d579489a94cbd0b6774e60bd7afffbea670d44
Merge: d2aec5d 44a1dd5
Author: Peter Goodhall <magicbug@users.noreply.github.com>
Date:   Sun Jan 31 16:49:54 2021 +0000

:

**Comment by M5JFS** [comment]
I don't have features like accumulated stats and time plotter in the analytics menu so assume i am not up to date

**Reply by magicbug** [reply]
That's definitely the latest commits, so you aren't seeing time plotter in the analytics dropdown? Your using an account with "administrator" privileges?

**Comment by M5JFS** [comment]
![image](https://user-images.githubusercontent.com/73586719/107118039-49e36100-6876-11eb-87c1-078f242918cf.png)

**Reply by magicbug** [reply]
That is seriously weird, you have Global Options in the Admin? With options to change language?

**Reply by magicbug** [reply]
Other thing is can you entertain checking on a different browser

**Reply by M5JFS** [reply]
freshly installed Opera to check your alternative browser suggestion    -  still the same
![image](https://user-images.githubusercontent.com/73586719/107119719-5325fb00-6881-11eb-8462-1c156cb3383f.png)

I like to be difficult

**Reply by magicbug** [reply]
That is really weird, you havent got two installs and the servers pointing to the wrong directory?

**Comment by M5JFS** [comment, marked as answer]
That sounds like a very reasonable suggestion. I'll check when I get home later.  I don't recall installing it more than once, but it's not beyond the realm of possibility.

**Reply by M5JFS** [reply]
Yes, that was the issue. thanks for you your help !

**Reply by magicbug** [reply]
Yay
